syntax = "proto3";
package RevaProtocol;
option java_package = "reva.protocol";
service RevaHeartbeat {
    rpc heartbeat(RevaHeartbeatRequest) returns (RevaHeartbeatResponse);
}

message RevaHeartbeatRequest {
    string message = 1;
}

message RevaHeartbeatResponse {
    string response = 1;
}